koopovereenkomst-v3-simple/pages/brk.tsx

```diff
@@ -0,0 +1,55 @@
+import { useSession } from "@inrupt/solid-ui-react";
+import { Button } from "@mui/material";
+import { Box } from "@mui/system";
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import ConnectSolid from "../src/ConnectSolid";
+import Layout from "../src/Layout";
+import VC, { VCType } from "../src/VC";
+
+export default function Brk() {
+
+    const { session } = useSession();
+    const [isLoggedOn, setIsLoggedOn] = useState(false);
+
+    const refreshState = () => {
+        setIsLoggedOn(session.info.isLoggedIn);
+        console.log("Refresh state. isLoggedOn: ", isLoggedOn);
+    }
+
+    useEffect(() => {
+        refreshState();
+        console.log("useEffect. isLoggedOn: ", isLoggedOn);
+    }, [])
+
+    return (
+        <Layout role="brk">
+            <Head>
+                <title>{"BRK"}</title>
+            </Head>
+            <Box sx={{
+                backgroundImage: "url('/solid-quest/images/mijnkadaster.png')",
+                backgroundRepeat: "no-repeat",
+                width: "100%",
+                height: "98rem",
+                display: "block",
+            }} onClick={refreshState}>
```

Comment:
Mijn 'oplossing' om state nogmaals te kunnen bijwerken ... :smirk:

Reply:
Prima! Enige alternatief wat ik kan bedenken in deze opzet is zodra je het extra tabje opent je kunt pollen op de vc locatie. Dan hoef je niet handmatig te refreshen. Voor nu helemaal prima op deze manier!